src/stan/analyze/mcmc/compute_potential_scale_reduction.hpp

```diff
@@ -16,6 +17,101 @@
 namespace stan {
 namespace analyze {
 
+inline double median( Eigen::MatrixXd d){
+    auto r { d.reshaped() };
+    std::sort( r.begin(), r.end() );
+    return r.size() % 2 == 0 ?
+        r.segment( (r.size()-2)/2, 2 ).mean() :
+        r( r.size()/2 );
+}
+
+Eigen::MatrixXd rankTransform(const Eigen::MatrixXd& matrix) {
```

Comment:
/**
 * Computes the blabla based on paper...
 *
 * @param matrix stores pointers to arrays of chains
 * @return potential scale reduction for the specified parameter
 */